CHANGELOG-1.1.md

```diff
@@ -1,10 +1,39 @@
+# TiDB Operator v1.1.0-beta.2 Release Notes
+
+This is a pre-release of `v1.1.0`, which focuses on the usability, extensibility and security of TiDB Operator. While we encourage usage in non-critical environments, it is **NOT** recommended to use this version in critical environments.
+
+## Changes since v1.1.0-beta.1
+
+## Action Required
+
+- `--default-storage-class-name` and `--default-backup-storage-class-name `are abandoned, storage class defaults to Kubernetes default storage class right now. If you have set default storage class different than Kubernetes default storage class, please set them explicitly in your tidb cluster helm or YAML files. ([#1581](https://github.com/pingcap/tidb-operator/pull/1581), [@cofyc](https://github.com/cofyc))
+
+## Other Notable Changes
+
+- Allow users to configure affinity and tolerations for `Backup` and `Restore` ([#1737](https://github.com/pingcap/tidb-operator/pull/1737), [@Smana](https://github.com/Smana))
+- Make AdvancedStatefulSet and Admission Webhook could work together. ([#1640](https://github.com/pingcap/tidb-operator/pull/1640), [@Yisaer](https://github.com/Yisaer))
+- Add a basic deployment example of TiDB cluster ([#1573](https://github.com/pingcap/tidb-operator/pull/1573), [@aylei](https://github.com/aylei))
```

Comment:
This needs to be clarified a bit. It's for the new CRD based cluster management.

Reply:
cc @aylei can you update the release note in your PR #1573? I'll regenerate this changelog later.

Reply:
updated